SSPSolution/GraphicsDLL/AnimationHandler.cpp

```diff
@@ -86,13 +76,15 @@ void AnimationHandler::Update(float dt)
 						/*If the player picks up the ball, go to player ball idle, otherwise regular idle.*/
 						if (m_AnimComponentList[m_AnimCompIndex]->source_State->stateIndex == PLAYER_PICKUP)
 						{
-							SetAnimationComponent(PLAYER_BALL_IDLE, 0.5f, SMOOTH_TRANSITION, true, false, 2.0f, 1.0f);
-							m_AnimComponentList[m_AnimCompIndex]->previousState = PLAYER_BALL_IDLE;
+							m_AnimComponentList[m_AnimCompIndex]->previousState = m_AnimComponentList[m_AnimCompIndex]->currentState;
+							SetAnimationComponent(PLAYER_BALL_IDLE, 0.55f, Blending::SMOOTH_TRANSITION, true, false, 0.9f, 1.0f);
```

Comment:
I would like that you add the enumeration name before the actual member just like you do with
`Blending::SMOOTH_TRANSITION`
In this case it would be
`AnimationStates::PLAYER_BALL_IDLE`

This is to imply that you use the AnimationStates enumeration and in some cases, it could avoid linker errors.